simulariumio/file_converter.py

```diff
@@ -82,5 +82,8 @@ def update_trajectory_info_version(self, data: Dict[str, Any]) -> Dict[str, Any]
                 "fovDegrees": DEFAULT_CAMERA_SETTINGS.FOV_DEGREES,
             }
             data["trajectoryInfo"]["version"] = 2
+        if int(data["trajectoryInfo"]["version"]) == 2:
+            # all the new fields from v2 to v3 are optional
+            data["trajectoryInfo"]["version"] = 3
         print(f"Updated TrajectoryInfo v1 -> v{self.current_trajectory_info_version}")
```

Comment:
this print statement should be in the v ==1 block.

Reply:
No I just shouldn't hardcode v1, fixed now